### PR TITLE
add entry_points, bump build in meta.yaml

### PR DESCRIPTION
**Pull request by NickleDave**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

@conda-forge-admin please rerender

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,9 +11,11 @@ source:
   sha256: b7bb621a47135e1d351becb82cdbe21237c39d4fcbbeaec677e31db02cc68900
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - vak=vak.__main__:main
```
Comment (bollwyvl) [inline recipe/meta.yaml:18]: for giggles, you may wish to add the following in `test:commands` (after `pip check`):

```yaml
- vak --version
- vak --help
```

Comment (NickleDave) [reply recipe/meta.yaml:18]: Ah right, that's what I should have done. Thank you

Comment (NickleDave) [reply recipe/meta.yaml:18]: Added another commit that does that

**Comment by NickleDave** [comment]
@bollwyvl thank you for the review

Should I just ignore the "I tried to re-render but there was nothing to do"?

Am I safe to merge?

**Comment by bollwyvl** [comment]
>  safe to merge?

Yep! Re-rendering only has an effect if:

- the CI needs to change (yours is up-to-date, so nothing to expect there)
- the README has to change
- other weird stuff (like global pins)

For example, you could add:

```yaml
extra:
  doc_url: https://vak.readthedocs.io
```

And then hit the render, and it would update the README.

You can also do it at home with:

```
conda smithy rerender
```

But to just solve the windows CLI issue, you're good to go.

**Comment by NickleDave** [comment]
I see. Thank you @bollwyvl you continue to be the best

**Pull request merged**